website/blog/2024-02-05-running-tabby-as-a-language-server/index.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: running-tabby-as-a-language-server
+title: "Running Tabby as a Language Server"
+authors: [icycodes]
+tags: [editor integration]
+---
+
+I am excited to announce that [Tabby Agent](https://github.com/tabbyml/tabby/tree/main/clients/tabby-agent/) now supports running as a [language server](https://microsoft.github.io/language-server-protocol/). This new feature allows you to enhance your code completion experience with Tabby in various text editors.
```

Comment:
Also mention and add a link to the npm package? https://www.npmjs.com/package/tabby-agent

Reply:
Updated in https://github.com/TabbyML/tabby/pull/1220/commits/a131d51d34937423688041cdb38d0600ed9e8416